src/status_im/contexts/wallet/events.cljs

```diff
@@ -375,3 +375,20 @@
             {:title   title
              :subject title
              :message content})]]}))
+
```

Comment:
not related to your pr, but this events file is starting to bloat a lot. 
We should probably split it up into some sections. 

Maybe we can add an issue to clean up any of the huge files in the wallet? 
e.g utils, this events file and let's see where else.

Reply:
I agree!